src/libs/Violations/ViolationsUtils.ts

```diff
@@ -371,6 +374,25 @@ const ViolationsUtils = {
             !TransactionUtils.hasReceipt(updatedTransaction) &&
             isControlPolicy;
 
+        // Check for itemized receipt requirement - policy level
+        const shouldShowItemizedReceiptRequiredViolation =
+            canCalculateAmountViolations &&
+            !isInvoiceTransaction &&
+            typeof categoryMaxAmountNoItemizedReceipt !== 'number' &&
+            typeof maxAmountNoItemizedReceipt === 'number' &&
+            expenseAmount > maxAmountNoItemizedReceipt &&
+            !TransactionUtils.hasReceipt(updatedTransaction) &&
+            isControlPolicy;
+
+        // Check for itemized receipt requirement - category level override
+        const shouldShowCategoryItemizedReceiptRequiredViolation =
+            canCalculateAmountViolations &&
+            !isInvoiceTransaction &&
+            typeof categoryMaxAmountNoItemizedReceipt === 'number' &&
+            expenseAmount > categoryMaxAmountNoItemizedReceipt &&
+            !TransactionUtils.hasReceipt(updatedTransaction) &&
+            isControlPolicy;
```

Comment:
Can we move the common conditions to a single place?

Reply:
I mean in the first one we are checking 7 things and in the second one we have 6 checks. Out of these, 4 are common between both, which can be consolidated.